anyway/widgets/widget_utils.py

```diff
@@ -160,3 +163,28 @@ def sort_and_fill_gaps_for_stacked_bar(
     res = fill_and_sort_by_numeric_range(data, numeric_range, default_order)
     res2 = second_level_fill_and_sort(res, default_order)
     return res2
+
+
+def get_involved_counts(selected_columns, start_year, end_year, severities, vehicle_types, location_info):
+    query = db.session.query(Involved) \
+        .with_entities(*selected_columns) \
+        .join(AccidentMarker, AccidentMarker.provider_and_id == Involved.provider_and_id) \

```

Comment:
As far as I remember we join both on the following 3 params: 
provider_code and accident_id and accident_year (see [ForeignKeyConstraint in Involved class](https://github.com/hasadna/anyway/blob/b20a204af080967c77fbec9ac2824b0b036391ab/anyway/models.py#L800))

Reply:
@ziv17, @tkalir  what do you think? Should we use the InvolvedView table or Original tables as @tkalir used?
Logic wise it should be the same.
Should we decide on convention? Any best practices we should follow?